jetty-ee9/jetty-ee9-servlet/src/main/java/org/eclipse/jetty/ee9/servlet/DefaultServlet.java

```diff
@@ -215,6 +232,40 @@ public void init()
             }
         }
 
+        if (getInitBoolean("allowAliases", true))
```

Comment:
This is a change of behaviour to EE8/EE9 so we probably should have this false by default.

Reply:
In Jetty 10 and Jetty 11 the DefaultServlet works with the Alias Checkers of the Context, no extra configuration required.
This default of true, is in line with that behavior.

Reply:
Reverted default back to `false`, but we should review that default with an eye on Jetty 10 / Jetty 11 behavior.